resolver/resolver.go

```diff
@@ -662,12 +663,30 @@ func (r *Resolver) resolveAutoBindFields(ctx *context, msg *Message) {
 		if autoBindField.Field.Type.Type != field.Type.Type {
 			continue
 		}
+		switch {
+		case autoBindField.ResponseField != nil:
+			autoBindField.ResponseField.Used = true
+		case autoBindField.MessageDependency != nil:
+			autoBindField.MessageDependency.Used = true
+		}
 		field.Rule = &FieldRule{
 			AutoBindField: autoBindField,
 		}
 	}
 }
 
+var nameRe = regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9_]*$`)
+
+func (r *Resolver) validateName(name string) error {
+	if name == "" {
+		return nil
+	}
+	if !nameRe.MatchString(name) {
+		return fmt.Errorf(`"%s" is invalid name. name should be in the following pattern: ^[a-zA-Z][a-ZA-Z0-9_]+$`, name)
```

Comment:
This is a different pattern from the regex defined above. Is this intentional? `*` => `+`

Reply:
Oh, It’s mistakes! I’ll fix it. Thank you 😄